src/pages/ReimbursementAccount/ValidationStep.js

```diff
@@ -27,12 +28,19 @@ import Section from '../../components/Section';
 import CONST from '../../CONST';
 import Button from '../../components/Button';
 import MenuItem from '../../components/MenuItem';
+import Enable2FAPrompt from './Enable2FAPrompt';
 
 const propTypes = {
     ...withLocalizePropTypes,
 
     /** Bank account currently in setup */
     reimbursementAccount: reimbursementAccountPropTypes,
+
+    account: PropTypes.shape({
```

Comment:
The guidelines mention: "All props are defined accurately and each prop has a /** comment above it */"